keepalived/meta/collectd.yml

```diff
@@ -0,0 +1,24 @@
+{%- if pillar.keepalived.cluster is defined %}
+{%- from "keepalived/map.jinja" import cluster with context %}
+{%- set vrrp_addresses = [] %}
+{%- for instance_name, instance in cluster.instance.iteritems() %}
+    {%- if instance.addresses is defined %}
+        {%- for address in instance.addresses %}
+            {%- set _ = vrrp_addresses.append({'label': instance_name, 'address': address}) %}
```

Comment:
You can use `do` instead. See http://jinja.pocoo.org/docs/dev/templates/#expression-statement.